src/scss/app.scss

```diff
@@ -0,0 +1 @@
+@import "~bootstrap";
```

Comment:
Do we have a double-import of Bootstrap here? We also imported it in `src/index.tsx`.